src/Domain/Entity/Event.php

```diff
@@ -21,6 +21,17 @@
 
 class Event
 {
+    const EVENT_STATUS_CONFIRMED = 'CONFIRMED';
```

Comment:
Instead of using constants, please use enums as shown here: https://github.com/markuspoerschke/iCal/blob/2.x/src/Domain/Enum/TimeZoneTransitionType.php